scalameta/parsers/js/src/main/scala/scala/meta/parsers/JSFacade.scala

```diff
@@ -59,7 +61,12 @@ object JSFacade {
       case _ => js.Dynamic.literal()
     }
```

Comment:
sure. By the way, I think we should ultimately have a "Tree.serialize" method in `contrib` that does all of these things.